lib/analysis_options.yaml

```diff
@@ -145,17 +150,20 @@ linter:
     prefer_expression_function_bodies: true
     avoid_escaping_inner_quotes: false
     prefer_for_elements_to_map_fromIterable: true # flutter
+    prefer_foreach: true
```

Comment:
I don't agree with this one. A `list.forEach()` does not support `return`, `break` and `continue` and can not be awaited. There is a `Future.forEach` but that is not as readable than a standard `for in`. Also, if you 'force' people to use the `forEach`, what they may or may not fully understand how it works and what the usecases for it are, there is a change that there will be some questionable design choices.

Reply:
set to false